### PR TITLE
Feature/issue 12 bot welcome message

### LINKED ISSUE #12: Bot welcome message
Implement a bot welcome message to greet new users, and tell about a purpose of the bot. The message should be the following (we may change it if needed, lets discuss in the comments below):

> Hello, {username}!
I am an event-maker Bot 🤖🎈I'll help you to create a calendar event which you may add to any calendar and easily share with friends! 😍
Click `Create a new event` button or send me a `/create` Command! 😊
Please note, I don't store and misuse any provided data 🔒

### PR DESCRIPTION
**Pull request by bmind12**
closes #12

### CODE REVIEW

File: src/Bot.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { Telegraf, Context } from 'telegraf';
+import i18next from 'i18next';
+
+export default class Bot {
```
Comment (fidelman) [inline src/Bot.ts:6]: just to clarify. Will we use default exporting through the project. AFAIK TS autoimport does not like it

File: src/Bot.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        this.bot = new Telegraf(token);
+    }
+
+    public start() {
+        this.bot.start(this.onStart);
+        this.bot.launch();
+    }
+
+    private onStart(ctx: Context) {
+        ctx.reply(i18next.t('welcome', { name: ctx.chat?.first_name }));
```
Comment (fidelman) [inline src/Bot.ts]: what about to add a default User Name?

File: package.json [hunk 3]
```diff
@@ -17,6 +17,7 @@
     },
     "dependencies": {
         "dotenv": "^8.2.0",
+        "i18next": "^19.8.2",
```
Comment (fidelman) [inline package.json:20]: 👍

**Review by fidelman — DISMISSED** [review]

File: src/Bot.ts (hunk 1 above)
Comment (bmind12) [reply src/Bot.ts:6]: It's the first time I hear of such issue. Do you have more details? I never experienced any problems with it.

Regarding what should we use, it should depend on a specific case.

File: src/Bot.ts (hunk 2 above)
Comment (bmind12) [reply src/Bot.ts]: Agree, will improve it

File: src/Bot.ts (hunk 1 above)
Comment (fidelman) [reply src/Bot.ts:6]: nono, it is okay, just double-checked it will work property with default importing as well

**Review by fidelman — APPROVED** [review]

**Pull request merged**